### PR TITLE
Feature/get user api integration

### PR DESCRIPTION
**Pull request by mohammadkhizerkhan**
**THIS PR POINTS TO:** [GET USER API ISSUE](https://github.com/Real-Dev-Squad/mobile-app/issues/189)

**THIS PR CONTAINS**

- Integrating redux-saga.
- Fetching user data and accessing globally using the redux store.
- Integrated get contribution API in profile screen

### CODE REVIEW

File: src/sagas/requests/fetchUser.ts
Comment (JAYATEJA04) [inline src/sagas/requests/fetchUser.ts:1]: Please add instagramUrl too.
@mohammadkhizerkhan

**Review by JAYATEJA04 — COMMENTED** [review]
requested a change

File: src/sagas/requests/fetchUser.ts
Comment (shreya-mishra) [reply src/sagas/requests/fetchUser.ts:1]: why do you need ig url ?

Comment (JAYATEJA04) [reply src/sagas/requests/fetchUser.ts:1]: A few people have shared their Instagram handle too, we can use that URL to display it on user data on the profile page.

Comment (shreya-mishra) [reply src/sagas/requests/fetchUser.ts:1]: @mohammadkhizerkhan  ??

**Review by shreya-mishra — APPROVED** [review]

**Review by shreya-mishra — APPROVED** [review]

**Pull request merged**